src/config-schemas/vitals-config-schema.ts

```diff
@@ -0,0 +1,83 @@
+import { validators } from "@openmrs/esm-module-config";
+
+export const vitalsConfigSchema = {
+  vitalsEncounterTypes: {
+    arrayElements: {
+      uuid: { validators: [validators.isString] },
+      description: { validators: [validators.isString] }
```

Comment:
Uh-oh! `description` is a reserved keyword for esm-module-config. See "[Defining a schema](https://github.com/openmrs/openmrs-esm-module-config#defining-a-schema)." What'll it be used for? Can we name the key something else?

Something I've considered is whether all the esm-module-config keywords -- `default`, `validators`, `arrayElements`, and `description`, should be prefixed, e.g. with an underscore. So config schemas would be writen like

```
vitalsEncounterTypes: {
  _arrayElements: {
    uuid: {
      _validators: [validators.isUuid /* doesn't exist yet but should exist! */],
      _description: "An encounter type UUID"
    },
    description: {
      _validators: [validators.isString],
      _description: "Information about this type of vitals encounter"
    }
  },
  default: []
}
```

Reply:
Is it a reserved prop on all levels or just for certain levels? Because on the given one I could see it as prop of the object defining `uuid`, but not in the object defining the `arrayElements`. (Just curious)

Reply:
Yeah, it's reserved on all levels. I think it would be strange to make an exception here -- "description a reserved keyword unless it's the child of arrayElements, in which case it behaves like a normal config key." Seems like it would make things more confusing.

Reply:
sounds good. Will changes it to 'name'.